src/pages/workspaces/workspace/analysis/Analyses.ts

```diff
@@ -215,7 +215,7 @@ const AnalysisCard = ({
     getFileName(name)
   ])
 
-  const toolIconSrc: string = Utils.switchCase(currentRuntimeToolLabel,
+  const toolIconSrc: string = Utils.switchCase(application,
```

Comment:
What exactly is the type of `application` here?  Is `currentRuntimeToolLabel` not the same?  I made this change recently, confused why `toolLabel` isn't matching on a `runtimeToolLabels`.  It seems worth looking into the below match cases to see if they can be changed to map to the better formed toolLabel types...

Reply:
well the issue is that currentRuntimeToolLabel is the label of the active runtime, so when you don't have an active runtime the logos are blank and when you have, say, an Rstudio runtime, all icons are set to Rstudio 
<img width="1481" alt="image" src="https://user-images.githubusercontent.com/85642387/235166287-26c756db-3ff4-4466-b063-ef7803ce6503.png">

Reply:
application is set in decorateAnalysisFiles and is set to `application: tools.RStudio.label` or jupyter depending on the extension